### PR TITLE
chore(param): new video params

### PR DESCRIPTION
**Pull request by gepifanio**
# Description

Fixes https://app.zenhub.com/workspaces/global-design-system-issues-tracker-603c265651b123001c916450/issues/unimelb/pattern-lib/1696

Two new params:

- `cc_load_policy=1`
- `modestbranding=1`

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] Make sure to do not repeat yourself (DRY)
- [x] Snapshots tested
- [x] A11y tested
- [x] CSS utilises BEM naming convention and structure
- [x] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [x] Crossbrowser testing (Safari 8+, iOS 8.4+, Android 4.4+, Firefox ESR (v52.x), iPhone (4s,6), iPad 2, Galaxy s5) 
- [x] My changes generate no new warnings
- [x] I have added tests (e2e and unit) that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Added the new component to the index.js (Vue and Lib) export

# Reviewer Checklist

- [ ] Code is not repeated (DRY)
- [ ] ES6+ code is used where possible
- [ ] Code does not mutate variables/objects/arrays but instead uses `map`, `filter` etc to return a new array/object
- [ ] CSS utilises BEM naming convention and structure
- [ ] Snapshots tested against a copy of dev snapshots to check UI changes are intended
- [ ] A11y tested
- [ ] Crossbrowser tested in at least IE11

### CODE REVIEW

**Review by jaredhenderson — COMMENTED** [review]
I think the script changes appear fine, but you might need to make it clearer in the documentation that:
- the `videoApi` is only for Youtube videos
- it's doing more than just adding analytics parameters

File: components/embed/stories/video.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ It accepts props such as:
 - `video`: Object for video, takes `url` (video embed URL), `duration` (in seconds) and an optional `preview` property for looping video
 - `orientation`: Can be 'portrait' or 'landscape', default to 'landscape'
 - `autoplay`: Boolean for autoplaying the looping video if one exists, otherwise only loops on mouseover
-- `videoApi`: Boolean to add additional query params for analytics
+- `videoApi`: *Youtube videos only* Boolean for to add additional query params to the `src`: `?autoplay=1&enablejsapi=1&rel=0&cc_load_policy=1&modestbranding=1&origin=https%3A%2F%2F[SUBDOMAIN].unimelb.edu.au`
```
Comment (jaredhenderson) [inline components/embed/stories/video.md:10]: Don't need the `for` here, should be `Boolean to add...` or `Boolean for adding...`

**Review by jaredhenderson — APPROVED** [review]
Just one minor typo, looks good otherwise

File: components/embed/stories/video.md (hunk 1 above)
Comment (gepifanio) [reply components/embed/stories/video.md]: Thanks for picking it up

**Comment by angieraj** [comment]
The preview link isnt really working for me, where can I test this @gepifanio

**Comment by gepifanio** [comment]
> The preview link isnt really working for me, where can I test this @gepifanio

VideoEmbed: https://deploy-preview-1700--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Embed&selectedStory=Video%20-%2016%2F9&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

VideoPlayer: https://deploy-preview-1700--pattern-lib-unimelb.netlify.app/?utm_source=github&utm_campaign=bot_dp&selectedKind=Embed&selectedStory=Video%20%28Today%29%20-%20Landscape&full=0&addons=1&stories=1&panelRight=0&addonPanel=REACT_STORYBOOK%2Freadme%2Fpanel

**Comment by angieraj** [comment]
Seems like the code is in both VideoEmbed and VideoPlayer but I can only see the actual captions on Video Player. Probably because the video on VideoEmbed have no captions at all. Can we add the MC video to the VideoEmbed to double check?

**Comment by angieraj** [comment]
Nice one thanks @gepifanio can see it working on both Embed and Player. Happy to proceed

**Pull request merged**